### PR TITLE
Use new !options to set window titles and code generation improvements

### PR DESCRIPTION
**Pull request by hbons**
Updated in both the app and extension, and also in the Figma component library.

### CODE REVIEW

**Comment by hbons** [comment]
example of generated code now:
image

File: FigmaSharp.NativeControls/FigmaSharp.NativeControls.Cocoa/FigmaBundleWindow.cs [hunk 1]
```diff
@@ -63,40 +63,56 @@ protected override void OnGetPartialDesignerClass (FigmaPartialDesignerClass par
 				var optionsNode = windowComponent.children.FirstOrDefault(s => s.name == "!options");
 				if (optionsNode is IFigmaNodeContainer figmaNodeContainer)
 				{
+					if (figmaNodeContainer.HasChildrenVisible("title"))
```
Comment (netonjm) [inline FigmaSharp.NativeControls/FigmaSharp.NativeControls.Cocoa/FigmaBundleWindow.cs:66]: I need improve this check to get the node as output

File: FigmaSharp.NativeControls/FigmaSharp.NativeControls/FigmaClassGenerator.cs [hunk 2]
```diff
@@ -89,7 +89,7 @@ protected override string OnGenerate ()
 
 	public abstract class FigmaClassBase
 	{
-		 protected const string InitializeComponentMethodName = "InitializeComponent";
```
Comment (netonjm) [inline FigmaSharp.NativeControls/FigmaSharp.NativeControls/FigmaClassGenerator.cs:92]: I tried to use same terms as code generation works in windows:
https://www.devcomponents.com/kb2/?p=1327

Comment (netonjm) [reply FigmaSharp.NativeControls/FigmaSharp.NativeControls/FigmaClassGenerator.cs:92]: I think all the designers Forms, WinForms, and Wpf uses InitializeComponent for this

Comment (hbons) [reply FigmaSharp.NativeControls/FigmaSharp.NativeControls/FigmaClassGenerator.cs:92]: ok!

**Comment by hbons** [comment]
ready to merge!

**Pull request merged**